static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js

```diff
@@ -3,21 +3,24 @@ import { getSync as geolocationGetSync } from 'lib/geolocation';
 import { acquisitionsBannerUsEoyTemplate } from 'common/modules/commercial/templates/acquisitions-banner-us-eoy';
 import { getArticleViewCountForWeeks } from 'common/modules/onward/history';
 
-// User must have read at least 5 articles in last 4 months (as 17 weeks)
+// User must have read at least 6 articles in last 4 months (as 17 weeks)
 const minArticleViews = 5;
 const articleCountWeeks = 17;
-const articleViewCount = getArticleViewCountForWeeks(articleCountWeeks);
+const articleViewCount = getArticleViewCountForWeeks(articleCountWeeks) - 1;
```

Comment:
shouldn't `minArticleViews` be 6 and `articleViewCount` remain the same?

Reply:
The minArticleViewCount is minimum number of articles that will be displayed in the copy, so we are indicating here to never show a lower number than 5. The articleViewCount is the number that is calculated for the reader, that would be rendered in their banner copy. Then we compare them in the canRun function to ensure the number that will be rendered is not lower than the minimum number.

Reply:
If we did it the way you suggested we could achieve the same result by making minArticleViews equal 6, then not subtract from the articleViewCount, however then we would need to render: `articleViewCount - 1` inside the string, where i don't think any calculations belong

Reply:
Given the worst thing we're doing is telling the reader they've read 1 less article than they actually have, this isn't a big enough issue to warrant any changes at this late stage, so I'll approve.

Reply:
no the copy says "You've read more than" which is the need to do this calculation

Reply:
Just been directed to the "You've read more than X", which said "You've read X articles" in the previous variant, so the -1 makes total sense now.